<launch>

  <node pkg="nao_custom" type="nao_behavior" name="nao_behavior" output="screen">
    <param name="NAO_IP" value="192.168.1.19"/>
    <param name="NAO_PORT" value="9559"/>
  </node>
  
</launch>
